packages/manager/modules/manager-components/src/task-tracker/task-tracker.template.html

```diff
@@ -5,8 +5,12 @@
                 <oui-modal
                     data-heading="{{:: $ctrl.heading }}"
                     data-primary-label="{{:: ('task_tracker_close' | translate) }}"
-                    data-primary-disabled="!$ctrl.isDone && !$ctrl.hasError"
-                    data-primary-action="$ctrl.onDone({ $tasks: $ctrl.tasks })"
+                    data-primary-disabled="!$ctrl.isClosable"
+                    data-primary-action="$ctrl.onDone({
+                        $tasks: $ctrl.tasks,
+                        $success: $ctrl.isDone,
+                        $error: $ctrl.errorMessage,
```

Comment:
why using `$` for each property ?

Reply:
Because these variables are intended to be used inside the `onDone: '&',` angular expression (a.k.a component binding), what I remember is a convention, right ? Like so
```javascript
data-on-done="$success ? doSomethingWith($tasks) : handle($error)"
```